pkg/providers/cloudstack/testdata/expected_results_main_no_machinetemplate_update_cp.yaml

```diff
@@ -389,18 +389,23 @@ metadata:
 spec:
   replicas: 3
   etcdadmConfigSpec:
-    etcdadmBuiltin: true
+    etcdadmBuiltin: false
```

Comment:
what does this do?

Reply:
https://github.com/mrajashree/etcdadm-bootstrap-provider/issues/14